simulation_parameters/SimulationParameters.cs

```diff
@@ -222,34 +222,48 @@ public GameCredits GetCredits()
         return gameCredits;
     }
 
-    public OrganelleDefinition GetRandomProkaryoticOrganelle(Random random)
+    public OrganelleDefinition GetRandomProkaryoticOrganelle(Random random, bool lawkOnly)
     {
         float valueLeft = random.Next(0.0f, prokaryoticOrganellesTotalChance);
 
-        foreach (var organelle in prokaryoticOrganelles)
+        // Filter to only LAWK organelles if necessary
+        var usedOrganelles = prokaryoticOrganelles;
+        if (lawkOnly)
+        {
+            usedOrganelles = usedOrganelles.Where(o => o.Lawk).ToList();
```

Comment:
This ToList seems to unnecessarily create a temporary list here. If you change `var` in used organelle to `IEnumerable<OrganelleDefinition>` I think this code will compile without the ToList call here.

Reply:
Attempting to do this is giving me a PossibleMultipleEnumeration error during style checking. Any suggestions or should I keep it as is?